### DISCUSSION TITLE
Installing SSL certificate for security onion

### DISCUSSION THREAD

**Discussion by AnilkumarITT** [Security-Onion-Solutions/securityonion#5117] [Unsupported Versions] [answered]
Hi Team 
When I access manager from web it will be non-secure version. I have certificate to be added but I am not able to see configuration file to add path of certificate. please let me know how to add certificate and make secure connection

Please help

thanks in advance

**Comment by facyber** [comment, marked as answer]
You can use LetsEncrypt for this:

1) Make a copy of `/etc/pki/managerssl.crt` and `/etc/pki/managerssl.key` files
2) Generate LE certificates
3) Copy `fullchain.pem` to `/etc/pki` folder as `managerssl.crt`
4) Copy `privkey.pem` to `/etc/pki` folder as `managerssl.key`
5) Restart Nginx with `so-nginx-restart`

This will setup LE TLS for your SO instance, and I suggest you to automate this process.
